app/uk/gov/hmrc/cipphonenumberstubs/services/responses/NotificationResponse.scala

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cipphonenumberstubs.services.responses
+
+object NotificationResponse {
+
+  val delivered = """
+                    |{
+                    |  "body": "cip-phone-service needs to verify your telephone number.\nYour telephone number verification code is HTNNQW.\nUse this code within 1 minutes to verify your telephone number.",
```

Comment:
i think the passcode should be dynamic in this version

Reply:
It will require a lot more refactoring and logic to make it dynamic if we want to do it the correct way. Lets discuss